src/pe_reports/data/database.ini

```diff
@@ -4,7 +4,9 @@ database=
 user=
 password=
 port=
-
+[peapi]
+api_key=
+api_url=
```

Comment:
We recommend alphabetizing lists whenever possible.  It looks like this file was alphabetized by section at one point.

Reply:
The `[peapi]` and `[postgres]` sections are now duplicated in this file- that should be corrected.

Also, `postgres` belongs alphabetically before `postgresql`, not after.

Reply:
commit f66cc14a691a07557f09f1451e5496ff3c007ce0 and 1713f77623e46206fc4f273282b3b3b776862abf

Reply:
Looks good now, thanks!